src/backend/distributed/commands/table.c

```diff
@@ -324,6 +324,9 @@ PostprocessCreateTableStmtPartitionOf(CreateStmt *createStatement, const
 		char *parentRelationName = generate_qualified_relation_name(parentRelationId);
 		bool viaDeprecatedAPI = false;
 
+		SwitchToSequentialAndLocalExecutionIfRelationNameTooLong(
```

Comment:
I don't really like the indentation in here. Sadly, I see no other option to fix this.